soc/arm/Kconfig

```diff
@@ -57,6 +57,30 @@ config NRF_SPU_RAM_REGION_SIZE
 	  RAM region size for the NRF_SPU peripheral
 endif
 
+if HAS_HW_NRF_MPU
+config NRF_MPU_FLASH_REGION_SIZE
+	hex
+	default 0x1000
```

Comment:
Just curious, why is this hard coded while the others are extracted from DT?

Reply:
Because it's different from the flash page size. NRF_MPU is for nrf51, which has flash page size of 0x400.